.pre-commit-config.yaml

```
@@ -43,67 +56,47 @@ repos:
       - id: rst-directive-colons
       - id: rst-inline-touching-normal
 
-  # Clean jupyter notebooks
-  - repo: https://github.com/srstevenson/nb-clean
-    rev: "3.1.0"
-    hooks:
-      - id: nb-clean
-  # Check for spelling
-  - repo: https://github.com/codespell-project/codespell
-    rev: "v2.2.6"
-    hooks:
-      - id: codespell
-        args: ["-L", "wille,linz,fro"]
-        exclude: "mqt/benchviewer/templates/legal.html"
-
-  # Format configuration files with prettier
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "v3.0.3"
-    hooks:
-      - id: prettier
-        types_or: [yaml, markdown, html, css, javascript, json]
-
+  # Python linting using ruff
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.1.4
+    rev: v0.4.5
```

```suggestion
    rev: v0.4.7
```
new ruff update from #2